docs/guides/custom-extensions.mdx

```diff
@@ -150,7 +150,8 @@ class MyExtension(SchemaExtension):
 
 - Execution
 
-`on_execute` can be used to run code on the execution step of the GraphQL
+`on_execute` can be used to run code on the execution step of the GraphQL. When
+using subscriptions, `on_execute` will be called for each subscription yield.
```

Comment:
That's a heavy question. I tried to look some references in other implementations though it seems like we are pioneers :upside_down_face: 
https://github.com/graphql-go/graphql/blob/f2b39caf7c92cab3f3f92726b55f59c74f54cfbc/subscription.go#L26

IMO a new hook might be better assuming `stream` and `defer` can benefit from it. otherwise I think `execute`
should suffice. 
Also I can't think of something you can do with the current `on_execute` that you can't do with `on_operation`  and no ready made extension uses it as well.
![image](https://github.com/user-attachments/assets/a84dbb08-3814-4529-83b9-462e0afa0b87)